src/Shared/SFA.DAS.SharedOuterApi/Services/LocationLookupService.cs

```diff
@@ -79,5 +79,12 @@ public async Task<LocationItem> GetLocationInformation(string location, double l
                 ? new LocationItem(location, getLocationsListItem.Location.GeoPoint, getLocationsListItem.Country) 
                 : null;
         }
+
+        public async Task<GetAddressesListResponse> GetExactMatchAddresses(string fullPostcode)
+        {
+            if (!Regex.IsMatch(fullPostcode, PostcodeRegex)) return null;
+
+            return await _locationApiClient.Get<GetAddressesListResponse>(new GetAddressesQueryRequest(fullPostcode, 1));
```

Comment:
please change the 1 to a suitable variable 'MinMatch' or similar

Reply:
Done